include/dem/pw_contact_force.h

```diff
@@ -129,6 +147,26 @@ class PWContactForce
       }
   }
 
+  /** This function is used to calculate the total force and total torque on
+   * each boundary
+   */
+  void
+  calculate_force_torque_on_boundary_function(const unsigned int boundary_id,
+                                              Tensor<1, dim>     add_force,
+                                              const Point<dim>   point_contact);
+
+  /** This function is used to initialize a map of vectors to zero
+   * with the member class boundary index which has the keys as information
+   */
+  std::map<unsigned int, Tensor<1, dim>>
+  initialize();
+
+  /** This function sums all the forces and torques from all the
+   * MPI processes
+   */
+  void
+  mpi_correction();
```

Comment:
Do you think this is a decent name for this function? Are we correcting something here?

Reply:
We are correcting a fake result that comes from mpi operations, so i thought that mpi_correction was a good idea but maybe too large in signification. @shahabgol , do you have a better idea like mpi_correction as suffix + force_torque_member ?